spec/models/ingredient_spec.rb

```diff
@@ -12,4 +33,10 @@
       it { should have_many(:recipes).through(:recipe_ingredients) }
    end
 
+   it "#sort_alphabetically" do
+      all_ingredients = Ingredient.all
+
+      expect(all_ingredients.sort_alphabetically).to eq([@garlic, @onion, @tomato])
+   end
```

Comment:
For this assertion, since it is a class method, you can just call it directly on the model and don't have to store it in a variable first.

```ruby
expect(Ingredient.sort_alphabetically).to eq([@garlic, @onion, @tomato])
```